tests/keys_test.py

```diff
@@ -0,0 +1,186 @@
+from datetime import datetime, timedelta
+from string import ascii_letters, digits
+from uuid import uuid4
+import json
+
+import pytest
+import keys
+
+config = json.load(open("botconf.json", "r"))
```

Comment:
Don't load a config (that might not exist on the test server); pass a known config.